README.md

```diff
@@ -81,7 +86,14 @@ Mimic the site's functionality around Users and Voting:
 - Add a list of voting users to the details page for each media
 - Don't allow a user to vote for the same media more than once
 - Add a page for each user, as well as a page showing a summary of all users
-- Make sure any models you've built so far are well-tested
+
+#### Testing
+**Before** moving on to Wave 3 you need to have:
+- Tests from Wave 1 passing
+- Presence and uniqueness validation tests for any new models
+- Relationship tests
+
+Focus on testing voting logic since this is the most complex part of Wave 2.
```

Comment:
It feels odd to me that this last line is not a bullet point.

Reply:
Should we include a list of test cases for voting? Something like:

- **Voting**
  - A user who has not already voted for a work can vote for that work
    - Even if they have voted for _other_ works
  - A user who has already voted for a work cannot vote for that work again
  - When a user votes for a work, it increases the work's vote count
- **Top 10**
  - Getting the top 10 works for a certain type of media only includes works of that type
  - Works are returned sorted by vote count
    - What happens in the case of a tie?
  - If there are less than 10 works of that type, all works are returned
  - If there are at least 10 works of that type, exactly 10 are returned

Reply:
This brings up an interesting point. The top 10 requirement is explicitly listed in wave 1, but won't be possible to implement until wave 2. Should this be moved?

Reply:
Additionally, is the learning goal of this assignment to practice writing model test syntax or coming up with the concepts of what to test. If its the former, then I think that level of detail makes sense to me. If its the latter, then it seems like this would be a giveaway.

Reply:
Moving the top-10 to wave 2 probably makes sense.

I think that, since this is the first time they've seen testing with Rails and we haven't done it for a month, I am more interested in them getting the syntax than in them coming up with the tests. But, maybe we could do an in-class activity on Wednesday to do a bit of design (talking about putting this logic in model methods) and come up with a list of test cases as a group.